core/src/test/java/com/sequenceiq/cloudbreak/service/image/ImageCatalogServiceTest.java

```diff
@@ -121,12 +122,18 @@
 
     private static final String CDP_DEFAULT_CATALOG_NAME = "cdp-default";
 
+    private static final String CATALOG_NAME = "test-catalog";
+
     private static final String CUSTOM_CATALOG_NAME = "custom-catalog";
 
     private static final String CUSTOM_BASE_PARCEL_URL = "https://myarchive.test.com";
 
     private static final String CUSTOM_IMAGE_ID = "customImageId";
 
+    private static final String UUID = java.util.UUID.randomUUID().toString();
```

Comment:
It is not needed to generate a real UUID, you can hardcode a static one into the USER_CRN or simply copy what we have in the EnvironmentApiConverterTest.